File: .claude/skills/calma/assets/lang/node/gen_fixture.js
// Calma cross-language fixture (Node.js).
const fs = require('fs');
fs.mkdirSync('runs', {recursive:true});
let lines = ['daily_return']; let acc = 1.0;
for (let i=0;i<252;i++){ const x = 0.0005 + 0.03*Math.sin(i*0.2); lines.push(String(x)); acc *= (1.0+x); }
fs.writeFileSync('runs/returns.csv', lines.join('\n')+'\n');
console.log('total_return='+(acc-1.0));

File: .claude/skills/calma/assets/lang/julia/gen_fixture.jl
# Calma cross-language fixture (Julia).
mkpath("runs")
r = [0.0005 + 0.03*sin(i*0.2) for i in 0:251]
open("runs/returns.csv","w") do io
    println(io, "daily_return"); for x in r; println(io, x); end
end
acc = prod(1.0 .+ r)
println("total_return=", acc-1.0)
